admin/admin.go

```diff
@@ -40,6 +50,16 @@ func (s *AdminServer) Start(ctx context.Context) error {
 	s.port = uint64(addr.Port)
 	s.log.Debug("admin server listening", "port", s.port)
 
+	// Set up RPC server
+	rpcServer := rpc.NewServer()
+	rpcMethods := &RPCMethods{Log: s.log}
+
+	if err := rpcServer.RegisterName("Admin", rpcMethods); err != nil {
+		return fmt.Errorf("failed to register RPC methods: %w", err)
+	}
+
+	s.log.Debug("admin HTTP server listening", "port", s.port)
```

Comment:
@jakim929 is this what you are expecting?, just wanted to confirm wrto setting up an rpc server